src/drivers/DriverManager.cpp

```diff
@@ -24,3 +24,21 @@ void DriverManager::activateAll() {
         drivers[i]->activate();
     }
 }
+
+void DriverManager::activateLast() {
+    int index = numberOfDrivers - 1;
+    drivers[index]->activate();
+}
+
+Driver* DriverManager::getDriverWithTag(char* tag) {
+    // Cycle through the drivers which have been added
+    for (int i = 0; i < numberOfDrivers; i++) {
+        // If we find a matching tag
+        if (drivers[i]->getDriverTypeTag() == tag) {
```

Comment:
Does this work? This looks like it's comparing the pointers and not the actual contents of the string.

Reply:
Maybe it would be better to use an enum?

Reply:
I thought about using an enum. We should open an issue for an optimization later.